### DISCUSSION TITLE
Quasar framework icon explorer

### DISCUSSION THREAD

**Discussion by s-spoon** [community/community#56534] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hi everyone. Now I'm using the nuxt3 + quasar framework for my application.
In my project I want to use the q-icon but I'm not sure the is the correct name of q-icon for each item.
Usually all libraries and frameworks have their icon explorer and so I can find the proper icon but now I can't find the q-icon explorer.
In the quasar guild, there is no page for q-icon.
I hope someone guide me in this problem. 
Thanks very much.

**Comment by silverspoon19931120** [comment, marked as answer]
Hi there! I'm happy to help. You are correct that Quasar does not have an official icon explorer for their q-icon component. However, there are a few resources you can use to help find the correct name of a Q-Icon.

Quasar Icons website - This is an unofficial website that lists all the available icons in Quasar and provides their names. You can search for an icon and see its name, as well as copy the code snippet to use it in your application: https://quasaricons.com/

Quasar documentation - The Quasar documentation has a list of built-in icons with their names and examples of how to use them in various components. You can check this page out: https://v3.quasar.dev/style/icon-fonts

Quasar GitHub repository - You can also browse the Quasar GitHub repository to find the SVG file for a specific icon. All the SVG files for Quasar icons are located in the "ui/icon-set/" directory of the repository. You can search for the desired icon name in the file names to locate the correct SVG file.

I hope these resources help you find the proper Q-Icon name for your project!
